gold-cc-cvc-input.html

```diff
@@ -40,7 +40,10 @@
 
   iron-icon {
     margin-left: 10px;
-    margin-bottom: 3px;
+  }
+
+  input[is="iron-input"] {
+    min-width: 0;
```

Comment:
Why don't we fix this in `<paper-input-container>` where the rest of the input styling is normalized?